classes/Absences/class.xaliAbsenceStatement.php

```diff
@@ -1,100 +1,109 @@
 <?php
-class xaliAbsenceStatement extends ActiveRecord {
 
-	const TABLE_NAME = 'xali_absence_statement';
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *********************************************************************/
```

Comment:
All license headers missing spaces (i assume this is a cs-fixer issue).

Pls use the current ILIAS 9 License header in all files.

Reply:
I don't see anything wrong with the license header?